contracts/contracts/GrantRegistry.sol

```diff
@@ -129,4 +129,23 @@ contract GrantRegistry {
     }
     return grantList;
   }
+
+  /**
+   * @notice Returns a boolean that is true when a given grant ID is in the registry
+   * @param _id Grant ID of a possible grant in the registry
+   */
+  function isGrantInRegistry(uint96 _id) public view returns (bool) {
+    Grant storage grant = grants[_id];
+    return grant.id == _id;
+  }
```

Comment:
Can we instead just `return _id  < grantCount`? Feels simpler, plus then you can move that logic into the GrantRound contract instead of needing a special method here

Reply:
Thinking about this a bit more, I think you're right and should just remove this function entirely, having the condition above cause a revert in the GrantRound contract